definitions/reports/inventory.rb

```diff
@@ -0,0 +1,92 @@
+# copy the file and add the .rb suffix
+module Checks
+  module Report
+    class Inventory < ForemanMaintain::Report
+      metadata do
+        description 'Facts about hosts and the rest of the inventory'
+      end
+
+      # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+      def run
+        # Hosts
+        hosts_by_type_count =
+          feature(:foreman_database).
+          query("select type, count(*) from hosts group by type").
```

Comment:
I think `Report` should have a `query` itself, no need to call `feature()` all the time

Reply:
nope, undefined method #query

Reply:
I said "should", not "does" :)
As in: we should implement a `Report.query` and use that, instead of calling `feature(:db).query` all the  time ;)

Reply:
https://github.com/ares/foreman_maintain/pull/15